### PR TITLE
Create client side assignment mock data

### PR DESCRIPTION
**Pull request by viivi-uhari**
Create client side assignment mock data to use on the client side (`mockAssignmentsClient.js`), and mock data of the format that may be gotten from the server side (`mockAssignmentsServer.js`). Fix accordions to match the format of `mockAssignmentsClient.js`.

I accidentally added changes on top of my local branch (edit-assignment) so you can ignore the commit messages and just look at the files changed. I removed the changes from the other branch and will push that branch later.

### CODE REVIEW

**Review by kiiamakir — APPROVED** [review]
Looks great!

File: client/src/services/formulas.js [hunk 1]
```diff
@@ -18,4 +19,11 @@ const setFormula = async (formulaInfo) => {
   return response.data;
 };
 
-export default { getFormulas, setFormula };
\ No newline at end of file
+// A temporary function to get a mock formula
+const getFormula = (formulaId) => {
+  const formula = mockFormulas.find( formula => formula.id === formulaId);
```
Comment (marillam) [inline client/src/services/formulas.js:24]: Space before last )? Not mandatory change imo

File: client/src/components/assignments/AssignmentCategory.js [hunk 2]
```diff
@@ -7,25 +7,28 @@ import PropTypes from 'prop-types';
...
+          ? <Typography align='left' variant='body2'>{'Formula: ' + formulasService.getFormula(formulaId)}</Typography>
+          : <></>
+        }
       </Box>
-      { subAssignments 
-        ? <CustomAccordion assignments={assignments} />
-        : <AccordionDetails out={true}><AssignmentText name={description} points={points} /></AccordionDetails>
+      { subAssignments.length !== 0
+        ? <CustomAccordion assignments={subAssignments} />
+        : <></>
```
Comment (marillam) [inline client/src/components/assignments/AssignmentCategory.js:31]: If no alternative (just <></>) then maybe 
{ subAssignments.length !== 0 &&
    Component here
}

File: client/src/components/assignments/Accordion.js [hunk 3]
```diff
@@ -89,14 +90,17 @@ const AssignmentText = ({ name, points }) => {
       columnGap: 3 
     }}>
       <Typography variant='body2'>{name}</Typography>
-      <Typography variant='caption' align='left'>{points + ' points'}</Typography>
+      { formulaId 
+        ? <Typography variant='caption' align='left'>{'Formula: ' + formulasService.getFormula(formulaId)}</Typography>
+        : <></>
+      }
```
Comment (marillam) [inline client/src/components/assignments/Accordion.js:96]: Switch syntax here?
{ formulaId &&
Component
}

**Review by marillam — CHANGES_REQUESTED** [review]
Looks very good, great job!

**Review by marillam — APPROVED** [review]

**Pull request merged**